### ISSUE TITLE
Installation fails on latest version of laravel and homestead 

### ISSUE THREAD

**Issue by robitwtms** [softonic/laravel-intelligent-scraper#16]
I've tried to install this package on the latest version of laravel, within a homestead environment, and it fails. Idk, maybe its something on my end, but I'm really not sure. I saw that this package has 2 dependencies, goutte and guzzle, both of which I tried installing but still no success. Idk, here is the output of the install: 
```
vagrant@homestead:~/project2$ composer require softonic/laravel-intelligent-scraper

    1/2:        http://repo.packagist.org/p/provider-latest$80d6a507ff378bcf1e002442bb8e921df3eb9c060920d4b52ea90ec1b9c66d31.json
    2/2:        http://repo.packagist.org/p/provider-2019-10$b39a019d0dd42a0491ba3c5537a01ed9a322512ed06a3dcdb39e4042ea509524.json
    Finished: success: 2, skipped: 0, failure: 0, total: 2
Using version ^0.0.16 for softonic/laravel-intelligent-scraper
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for softonic/laravel-intelligent-scraper ^0.0.16 -> satisfiable by softonic/laravel-intelligent-scraper[0.0.16].
    - Conclusion: remove laravel/framework v6.9.0
    - Conclusion: don't install laravel/framework v6.9.0
    - softonic/laravel-intelligent-scraper 0.0.16 requires illuminate/events ^5.8 -> satisfiable by illuminate/events[5.8.x-dev, v5.8.0, v5.8.11, v5.8.12, v5.8.14, v5.8.15, v5.8.17, v5.8.18, v5.8.19, v5.8.2, v5.8.20, v5.8.22, v5.8.24, v5.8.27, v5.8.28, v5.8.29, v5.8.3, v5.8.30, v5.8.31, v5.8.32, v5.8.33, v5.8.34, v5.8.35, v5.8.36, v5.8.4, v5.8.8, v5.8.9].
    - don't install illuminate/events 5.8.x-dev|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.0|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.11|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.12|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.14|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.15|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.17|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.18|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.19|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.2|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.20|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.22|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.24|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.27|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.28|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.29|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.3|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.30|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.31|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.32|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.33|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.34|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.35|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.36|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.4|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.8|don't install laravel/framework v6.9.0
    - don't install illuminate/events v5.8.9|don't install laravel/framework v6.9.0
    - Installation request for laravel/framework (locked at v6.9.0, required as ^6.2) -> satisfiable by laravel/framework[v6.9.0].


Installation failed, reverting ./composer.json to its original content.
```
If you have any ideas please let me know.

**Comment by joskfg** [comment]
Sorry for answering so late, we will answer in a shorter time from now on.

The issue is produced because this package uses laravel 5.8 dependencies. I didn't try it with laravel 6.x but if you can test it, we should be able to update the dependencies to accept laravel 6.x dependencies.

If you can't try it, we will do as best effort.

Thanks for your report

**Comment by joskfg** [comment]
Fixed by commit 2fd8449d591742cf8ab7b98f2ca314fb5829ba32
